cve_bin_tool/helper_script.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import sqlite3
+
+from cve_bin_tool.cvedb import DBNAME, DISK_LOCATION_DEFAULT
+from cve_bin_tool.extractor import Extractor
+from cve_bin_tool.log import LOGGER
+
+
+class HelperScript:
+    """Helps Contributors to understand how various common filenames, version-strings, etc. look"""
+
+    dbname = os.path.join(DISK_LOCATION_DEFAULT, DBNAME)
+
+    def __init__(self, filename, vendor_name=None, product_name=None):
+        self.extractor = Extractor()
+        self.logger = LOGGER.getChild(self.__class__.__name__)
+        self.product = product_name or self.parse_filename(filename)
```

Comment:
Sometimes, finding the product_name is not feasible (ex: tomcat10-10.0.6-1.el8.harbottle.x86_64.rpm). Here, parsing it gives us "tomcat10", which is not correct. So, I am also taking the product name as an input, in this case it would be "tomcat". But, in most of the cases, this wouldn't be required.